easybuild/easyconfigs/p/PHYLIP/PHYLIP-3.697-GCC-12.3.0.eb

```
@@ -0,0 +1,27 @@
+easyblock = 'MakeCp'
+
+name = 'PHYLIP'
+version = '3.697'
+
+homepage = 'https://evolution.genetics.washington.edu/phylip'
+description = "PHYLIP is a free package of programs for inferring phylogenies."
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+# -fcommon is required to avoid 'multiple definitions' linking error
+toolchainopts = {'extra_cflags': "-fcommon"}
+
+source_urls = ['https://evolution.gs.washington.edu/phylip/download/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['9a26d8b08b8afea7f708509ef41df484003101eaf4beceb5cf7851eb940510c1']
```

I'm also getting a checksum error locally, and when inspecting the file it's basically an HTML page saying that their site has been moved.
```suggestion
source_urls = ['https://phylipweb.github.io/phylip/download/']
sources = [SOURCELOWER_TAR_GZ]
checksums = ['9a26d8b08b8afea7f708509ef41df484003101eaf4beceb5cf7851eb940510c1']
```